craigslist/base.py

```diff
@@ -357,7 +357,7 @@ def parse_attrs(self, result):
         attrs_after_colon = set(
             attr.split(': ', 1)[-1] for attr in result['attrs'])
         for key, options in iteritems(self.get_list_filters(self.url)):
-            for option in options['value']:
+            for option in list(options['value'].keys()):
```

Comment:
No need to cast to list.

```suggestion
            for option in options['value'].keys():
```